demo/views/demo-all.dust

```diff
@@ -34,14 +35,11 @@
   </div>
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
It does, but I couldn't get it to work properly after updating from bluemix-components to carbon-components. So I need to figure out how to use the demo.js file without things failing.